src/search/FindUtils.js

```diff
@@ -35,10 +35,29 @@ define(function (require, exports, module) {
         StringUtils = require("utils/StringUtils"),
         _ = require("thirdparty/lodash");
 
-    var instantSearchDisabled = false,
-        indexingInProgress = false,
-        workerSearchCount = 0,
-        collapseResults = false;
+    /**
+     * if instant search is disabled, defaults to false
+     * @type {boolean}
```

Comment:
These variables are not exported and should be `@private` right?

Reply:
That's right. Missed it!